### PR TITLE
Fix check for custom komi.

### PR DESCRIPTION
**Pull request by GreenAsJade**
Simple bugfix so that custom komi of 0 gets the warning, like any other custom komi.

### CODE REVIEW

File: src/components/TimeControl/util.ts [hunk 1]
```diff
@@ -433,7 +433,7 @@ export function usedForCheating(time_control) {
         case "byoyomi":
             return !(
                 time_control.main_time > QUESTIONABLE_ABSOLUTE_TIME ||
-                time_control.period_time > QUESTIONABLE_SECONDS_PER_MOVE
```
Comment (anoek) [inline src/components/TimeControl/util.ts]: Isn't this going to flag pretty much every live game as questionable? It's quite common to have say 5x30s byoyomi, which if i'm not mistaken, would be flagged as questionable here if the main time wasn't at least 15 minutes

File: src/views/Play/Play.tsx [hunk 2]
```diff
@@ -591,6 +591,9 @@ export class Play extends React.Component<PlayProperties, any> {
 
         let challengeList = isLive ? this.state.live_list : this.state.correspondence_list;
 
+        challengeList.forEach(element => {
+            console.log("komi: " + element.komi);
```
Comment (anoek) [inline src/views/Play/Play.tsx]: Probably don't need this log anymore for production

File: src/components/TimeControl/util.ts [hunk 3]
```diff
@@ -433,7 +433,7 @@ export function usedForCheating(time_control) {
...
+                time_control.period_time * time_control.periods > QUESTIONABLE_ABSOLUTE_TIME
```
Comment (anoek) [inline src/components/TimeControl/util.ts]: Isn't this going to flag pretty much every live game as questionable? It's quite common to have say 5x30s byoyomi, which if i'm not mistaken, would be flagged as questionable here if the main time wasn't at least 15 minutes

File: src/views/Play/Play.tsx (hunk 2 above)
Comment (GreenAsJade) [reply src/views/Play/Play.tsx]: Oh yeah, I will (or you can) get rid of that :)

File: src/components/TimeControl/util.ts (hunk 3 above)
Comment (GreenAsJade) [reply src/components/TimeControl/util.ts]: No, I think this is one of these upside down logic things.

This time setting is used for cheating if it is _not_ the case that those OR conditions are true.

So this setting is used for cheating 
```
if  (it is not the case that
      (main time is big enough or 
        there is enough time in the periods))

```

Comment (GreenAsJade) [reply src/components/TimeControl/util.ts]: Oh wait, you are right.

Hmm - what was I thinking!

I should have just stuck to fixing the actual bug!

**Pull request merged**